### ISSUE TITLE
Home Screen on Mobile

### ISSUE THREAD

**Issue by Ernie3** [KHS-Minithon/website#34]
The mobile look a bit clunky. Maybe we can adjust some of the heights. But if we do not get to this by mid week (before it gets pushed to prod), no biggie.

![capture](https://cloud.githubusercontent.com/assets/6378488/10718874/0cc5da98-7b53-11e5-9708-7cf5c4eb55e2.jpg)

**Comment by theseanconn** [comment]
"Mini-THON" should be on one line

On Sunday, October 25, 2015, Ernie Wilson <notifications@github.com> wrote:

> The mobile look a bit clunky. Maybe we can adjust some of the heights. But
> if we do not get to this by mid week (before it gets pushed to prod), no
> biggie.
>
> [image: capture]
> <https://cloud.githubusercontent.com/assets/6378488/10718874/0cc5da98-7b53-11e5-9708-7cf5c4eb55e2.jpg>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KHS-Minithon/website/issues/34>.
>


-- 
~Sean

**Comment by Ernie3** [comment]
I agree. It's okay for now I think, want to get done all high and medium tickets first. If it gets pushed to prod, it's not the end of the world. But we will address this.

**Comment by Ernie3** [comment]
More current:

![capture](https://cloud.githubusercontent.com/assets/6378488/10776241/05c55f18-7ce6-11e5-9bd5-89e63202aa35.jpg)

**Comment by JoseRoman** [comment]
I will take over this
